test/test-experiment-recipe-firefox.ts

```diff
@@ -0,0 +1,47 @@
+import {typeGuards} from "..";
+
+const TEST_EXPERIMENT = {
+  "id": "ABOUTWELCOME-PULL-FACTOR-REINFORCEMENT-76-RELEASE",
+  "enabled": true,
+  foo: "s",
```

Comment:
AH, this is neat. I didn't get around to testing the schemas with unexpected properties yet (to see if AJV currently ignores, discards, or throws).